platform/viewer/cypress/support/commands.js

```diff
@@ -44,7 +44,8 @@ Cypress.Commands.add('openStudy', patientName => {
   cy.wait('@getStudies');
   cy.get('#studyListData .studylistStudy', { timeout: 5000 })
     .contains(patientName)
-    .first().click();
+    .first()
+    .click({ force: true });
```

Comment:
I was getting some E2E errors while trying to find and click in a study. The study was being displayed but click was not working. `{ force: true }` solved this problem.

Reply:
That's odd. I would have guessed it was because the `contains` should be wrapped in a should so it can leverage the timeout in the above `cy.get`. I wonder what's going on here that requires us to force a click. That usually means the found element wouldn't be clickable by the user (off screen, invisible, etc.)